utils/supported-browsers-from-mdn.mjs

```diff
@@ -49,6 +49,10 @@ function getBrowsersFromFeature(mdnConfigPath, feature) {
 					return false;
 				}
 
+				if (browserEntry.partial_implementation) {
+					return false;
+				}
```

Comment:
I overlooked that we didn't have an explicit check for this yet.
Was there a reason I am not currently thinking of?

The only difference I noticed is for `color()` Safari 10.x had support for `color(srgb ...)` and `color(display-p3 ...)` but not `xyz`, `rec2020`, ...

So I think this is a pure fix?

Reply:
I agree, probably an overlook